### ISSUE TITLE
Error in template strings: Emits strange $

### ISSUE THREAD

**Issue by madskonradsen** [decaffeinate/decaffeinate#230]
POC: http://decaffeinate.github.io/decaffeinate/repl/#?evaluate=true&code=cssVar%20%3D%20%22%22%22%0A%20%20.some-class%20%7B%0A%20%20%20%20content%3A%20data-uri(%5C%22%23%7BsomeObj%7D%5C%22)%3B%0A%20%20%7D%0A%22%22%22

The above example is valid coffeescript.

**Comment by madskonradsen** [comment]
The original issue seems to be fixed, but another issue arises

```
 cssVar = """
  .some-class {
    content: data-uri(\"#{someObj}\");
  }
"""
```
gets translated to:
```
let cssVar = `.some-class {
$  content: data-uri(\"#{someObj}\");
}`;
```
Notice the strange `$`

**Comment by eventualbuddha** [comment]
This was a duplicate of #212, which is fixed in v2.11.1.
